webapp/portlet/src/main/webapp/spaces-administration/components/drawer/ExoGroupBindingSecondLevelDrawer.vue

```diff
@@ -307,119 +306,10 @@ export default {
         boundGroupIds.forEach(groupId => {
           boundItems.push(this.getItem(groupId));
         });
+        boundItems.forEach(item => item.bound = true);
```

Comment:
Could you merge both of those loops in one

Reply:
Okay